quic-forward-proxy/src/outbound/tx_forward.rs

```diff
@@ -138,16 +144,26 @@ pub async fn tx_forwarder(
                             break 'tx_channel_loop;
                         }
 
+                        // micro-batch
+                        // - after receiving one message wait a bit before draining the channel
+                        // - should be a fraction of slot time - i.e. small enough to not miss the slot
+                        // - check prometheus metric 'literpcproxy_batch_size'
+                        sleep(MICROBATCH_WINDOW).await;
+
                         let mut transactions_batch: Vec<Vec<u8>> = packet.transactions.clone();
 
-                        'more: while let Ok(more) = per_connection_receiver.try_recv() {
+                        'drain_loop: while let Ok(more) = per_connection_receiver.try_recv() {
```

Comment:
why do you need to specify drain_loop, seems like goto which is not required right ?

Reply:
it's not needed but ready more clearly IMO

Reply:
imo this is a bad design. if u need labels to break out loops. could be done better

Reply:
no I do not need them.

compare it with this - the I scan that code I constantly look for the scope of breaks:
<img width="762" alt="image" src="https://github.com/blockworks-foundation/lite-rpc/assets/95291500/207c5a6d-4df0-4381-b2e2-1a17bb0b7202">

Reply:
I was staring a bit on the code, but so little room for improvement - can you suggest something?

Reply:
I guess I read more clearly when i read continue and break logic.